packages/polaris-viz/src/components/Grid/components/Tooltip.tsx

```diff
@@ -1,54 +1,68 @@
+import {forwardRef} from 'react';
+import {createPortal} from 'react-dom';
+
 import type {CellGroup} from '../types';
+import {useRootContainer} from '../../../hooks/useRootContainer';
 
 import styles from './Tooltip.scss';
 
+const TOOLTIP_ID = 'polaris_viz_tooltip_root';
```

Comment:
Can we extract this out into a shared constant that `packages/polaris-viz/src/components/TooltipWrapper/TooltipWrapper.tsx` can also use?